api/routes/users/user.ts

```diff
@@ -96,19 +95,22 @@ export const registerUser = async (req: Request, res: Response) => {
       return res.status(401).send('User already exist');
     }
 
-
     //3. Bcrypt the user password
     const salt = await bcrypt.genSalt(10);
     const bcryptPassword = await bcrypt.hash(password, salt);
 
     //4. enter the new user inside our db
-   
+
     const newUser = await pool.query('INSERT INTO users (name, email, password) VALUES ($1, $2, $3) RETURNING *', [
       name,
       email,
       bcryptPassword,
     ]);
- 
+
+    ['przeczytane', 'do przeczytania', 'chcę przeczytać'].map(async (s) => {
```

Comment:
Fajnie by było zaciągnąć to z innego pliku constants, że jak nam się zmieni że chcemy mieć kolejną kategorię to móc to ustawiać wszystko w jednym pliku